X16 Reference - 06 - Math Library.md

```diff
@@ -31,30 +31,30 @@ The following calls are new to the X16 and were not part of the C128 math librar
 
 | Address | Symbol   | Description                                     |
 |---------|----------|-------------------------------------------------|
-| $FE87  | `FLOAT`  | FAC = (s8).A   convert signed byte to float     |
-| $FE8A  | `FLOATS` | FAC = (s16)facho+1:facho                        |
-| $FE8D  | `QINT`   | facho:facho+1:facho+2:facho+3 = u32(FAC)        |
-| $FE93  | `FOUTC`  | Convert FAC to ASCIIZ string at fbuffr - 1 + .Y |
+| $FE87  | `FLOAT`  | FACC = (s8).A   convert signed byte to float     |
+| $FE8A  | `FLOATS` | FACC = (s16)facho+1:facho                        |
+| $FE8D  | `QINT`   | facho:facho+1:facho+2:facho+3 = u32(FACC)        |
+| $FE93  | `FOUTC`  | Convert FACC to ASCIIZ string at fbuffr - 1 + .Y |
 
 ## Movement
 
 | Address | Symbol   | Description                          |
 |---------|----------|--------------------------------------|
-| $FE5A  | `CONUPK` | move RAM MEM to ARG                  |
-| $FE5D  | `ROMUPK` | move ROM/RAM MEM to ARG (use CONUPK) |
-| $FE60  | `MOVFRM` | move RAM MEM to FACC (use MOVFM)     |
-| $FE63  | `MOVFM`  | move ROM/RAM MEM to FACC             |
-| $FE66  | `MOVMF`  | move FACC to RAM MEM                 |
-| $FE69  | `MOVFA`  | move ARG to FACC                     |
-| $FE6C  | `MOVAF`  | move FACC to ARG                     |
+| $FE5A  | `CONUPK` | ARG = UNPACK(RAM MEM)                |
```

Comment:
should define somewhere what "UNPACK" means, otherwise it will just be more confusing?